### ISSUE TITLE
Switch template: $role ignored in a coa scope

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5164]
**Describe the bug**
When you use a switch template with `radiusDisconnect = coa` and a `coa` scope that use `$role` variable, you get an error.

**To Reproduce**
Steps to reproduce the behavior: same as for #5123 but with following switch template:
```
description = Nico NG
radiusDisconnect = coa

acceptVlan = <<EOT
Tunnel-Medium-Type = 6
Tunnel-Type = 13
Tunnel-Private-Group-ID = $vlan
EOT

acceptRole = <<EOT
Filter-ID = $role
EOT

coa = <<EOT
Calling-Station-Id = ${macToEUI48($mac)}
User-Name = $last_accounting.username
NAS-IP-Address = $disconnectIp
NAS-Filter-Rule = $role
EOT
```
Switch config:
```
[192.168.0.6]
description=bar
type=Aruba::Nico
RoleMap=Y
VlanMap=N
registrationRole=reg
```

Error message when you try to reevaluate access:
```
Feb 27 14:12:22 pfcen7night pfqueue: pfqueue(8231) WARN: [mac:77:77:77:77:77:77] Use of uninitialized value in join or string at /usr/local/pf/lib/pf/mini_template.pm line 39.
 (pf::mini_template::process)
Feb 27 14:12:22 pfcen7night pfqueue: Use of uninitialized value in join or string at /usr/local/pf/lib/pf/mini_template.pm line 39.
Feb 27 14:12:22 pfcen7night pfqueue: Unknown RADIUS tuple NAS-Filter-Rule =>
Feb 27 14:12:22 pfcen7night pfqueue: at /usr/local/pf/lib/pf/util/radius.pm line 136.
Feb 27 14:12:22 pfcen7night pfqueue: pfqueue(8231) WARN: [mac:77:77:77:77:77:77] Unknown RADIUS tuple NAS-Filter-Rule => 
 at /usr/local/pf/lib/pf/util/radius.pm line 136.
 (Carp::carp)
```
`NAS-Filter-Rule` attribute is not present in CoA-Request sent to network device.

**Expected behavior**
`NAS-Filter-Rule` attribute should be present in CoA-Request with value defined in switch configuration.

**Comment by jrouzierinverse** [comment]
The real problem is that NAS-Filter-Rule is not defined in ```/usr/local/pf/lib/pf/util/dictionary```
The proper solution is to parse the freeradius dictionary in perl.
The quick solution is to add NAS-Filter-Rule to /usr/local/pf/lib/pf/util/dictionary.
I will add the quick solution in devel and maintence and add the proper solution at a later date.
As the proper solution will not cleanly commit in maintence.

**Comment by nqb** [comment]
Next part in #5168

**Comment by nqb** [comment]
@jrouzierinverse, any reason this fix is only in maintenance/9.3 and not in maintenance/9.2 ?
